src/data/faq_de.json

```diff
@@ -48,14 +48,17 @@
-                            "Siehe auch den Blog-Artikel <a href='../blog/2021-05-12-corona-warn-app-version-2-2//' target='_blank' rel='noopener noreferrer'>Corona-Warn-App Version 2.2 ermöglicht die Erstellung von Schnelltest-Profil und Fehlerbericht</a>."
+                            "Mit Version 2.2 bietet die Corona-Warn-App (CWA) eine Funktion zur technischen Problem-Analyse. Fehlerberichte sollen den Entwicklern helfen, schneller eine Ursachen-Analyse zu erstellen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei dem technischen Support, entweder per Telefon (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) oder per E-Mail (<a href='mailto:corona-warn-app.opensource@sap.com'>corona-warn-app.opensource@sap.com</a>), um zu prüfen, ob es ggf. schon eine Lösung für das Problem gibt. Ein Fehlerbericht wird nur benötigt, wenn der technische Support danach fragt.",
+                            "In diesem Fall stehen Ihnen verschiedene Möglichkeiten zur Übermittlung der ID des Fehlerberichts zur Verfügung:",
+                            "<ul><li>Öffnen Sie einen Issue auf GitHub (<a href='https://github.com/corona-warn-app/cwa-app-android/issues/new/choose' target='_blank' rel='noopener noreferrer'>Android</a> oder <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/new/choose' target='_blank' rel='noopener noreferrer'>iOS</a>) und nennen die ID Ihres Fehlerberichts </li><li> Teilen Sie die ID dem technischen Support mit </li><li> Schreiben Sie eine Bewertung zur Corona-Warn-App im Apple App Store bzw. Google Play Store und nennen die ID Ihres Fehlerberichts</ul></li>",
+                            "Egal für welche Kontaktmöglichkeit Sie sich entscheiden, die nachfolgenden Hinweise tragen dazu bei, dass Ihr Problem so schnell wie möglich gelöst wird:",
+                            "<ol><li>Beschreiben Sie das Problem so genau wie möglich</li><li>Starten Sie die Fehleranalyse in der CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>siehe Blog-Post</a>)<br>- Versuchen Sie das fehlerhafte Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen<br>- Ist das gelungen, senden Sie den Fehlerbericht an den CWA-Server. Die Daten werden dabei anonymisiert und verschlüsselt. Bei Interesse können Sie den Bericht auch lokal auf Ihrem Smartphone abspeichern<br>- Als Rückmeldung erhalten Sie eine eindeutige Kennung, die Sie in der Liste aller IDs finden <br>- Stoppen Sie die Fehleranalyse</li><li>Teilen Sie die ID dem technischen Support mit</li></ol>",

```

Comment:
Maybe it would be better to use an unordered list (`ul`) for this, instead of minus signs? In the screenshot, you can see that it does not line-wrap nicely otherwise.

Reply:
@fynngodau 

Thanks for the suggestion, I'll take a look at this once a decision on my proposal was made. 🙂

Reply:
I just tried this and this does not work easily here: 
<img width="671" alt="Bildschirmfoto 2021-05-19 um 22 00 57" src="https://user-images.githubusercontent.com/67682506/118876568-b21df600-b8ed-11eb-80c7-84b8117e39ba.png">


Any ideas @fynngodau @MikeMcC399?

Reply:
It looks like you didn't quite get it right. Maybe you made a mistake with nesting?

This line works:

    <ol><li>Describe the problem as accurately as possible</li><li>Start the error analysis in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog post</a>)<ul><li>Try to reproduce the incorrect behaviour or to generate the error message</li><li>If this is successful, send the error report to the CWA server. The data will be anonymised and encrypted. If you are interested, you can also save the report locally on your smartphone</li><li>You can then find your unique ID on the bottom of the 'Error Reports' screen, scroll down and click on 'ID History'</li><li>Stop the error analysis</li></ul></li><li>Share the 'unique report ID' with the technical support</li></ol>

![image](https://user-images.githubusercontent.com/16943720/118924094-e53ba600-b93c-11eb-9c9a-6582aecb95df.png)

Reply:
Yeah, I did it wrong and forget `<li>`, thank you!

Reply:
Added in https://github.com/corona-warn-app/cwa-website/pull/1267/commits/8afaa6284a967a2a9542ace1ce56c17451b560f2